week3/assignment/src/components/header/Header.jsx

```diff
@@ -0,0 +1,102 @@
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+const HeaderContainer = styled.header`
+  display: flex;
+  justify-content: space-between; 
+  align-items: center;
+  padding: 1.5rem;
+  background-color: ${(props) => props.theme.colors.black};
+`;
+
+const LeftSection = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+const RightSection = styled.div`
+  display: flex;
+  gap: 1rem;
+  align-items: center;
+`;
```

Comment:
p1: 현재 게임이 시작되어서 타이머가 실행되면 타이머 숫자가 바뀜에 따라 너비가 달라지기 때문에 `RightSection`영역 내부 요소인 `LevelSelect`와 `Timer` 부분이 계속해서 좌우로 흔들려서?보이는 사용자 경험 측면에서 좋지 않은 문제가 발생하고 있는데요. 다음과 같이 `RightSection`에 넉넉한 `width`값을 지정해주면 해당 이슈 해결할 수 있을거 같아요!
```suggestion
const RightSection = styled.div`
  display: flex;
  width: 10rem;
  gap: 1rem;
  align-items: center;
`;
```